x-pack/test/observability_functional/apps/observability/pages/alerts/custom_threshold.ts

```diff
@@ -0,0 +1,112 @@
+          sort_field: 'name',
+          sort_order: 'asc',
+          filter_consumers: ['apm', 'infrastructure', 'logs', 'uptime', 'slo', 'observability'],
+        });
+      expect(rules.data.length).toEqual(1);
+      expect(rules.data[0]).toEqual(
+        expect.objectContaining({
+          name: 'test custom threshold rule',
+          tags: ['tag1'],
+          params: expect.objectContaining({
+            alertOnGroupDisappear: true,
+            alertOnNoData: true,
+            criteria: [
+              {
+                comparator: '>',
+                label: 'test equation',
+                metrics: [
+                  {
+                    aggType: 'avg',
+                    field: 'metricset.rtt',
+                    name: 'A',
+                  },
+                  {
+                    aggType: 'count',
+                    filter: 'service.name : "opbeans-node"',
+                    name: 'B',
+                  },
+                ],
+                threshold: [200, 250],
+                timeSize: 2,
+                timeUnit: 'd',
+              },
+            ],
+            groupBy: ['docker.container.name'],
+            // searchConfiguration: {
```

Comment:
One thing I'm not sure how to test is the data view configuration. I didn't see anywhere in the rule definition that directly referenced the data view? Or is this index id below someone a uuid referencing the data view?

Reply:
I know this searchConfiguration is passed to the executor a bit differently but I didn't find a way to see how we fetch it in the edit flyout, I will check more and will comment here if I find out something.